### ISSUE TITLE
The circles aren't centered properly

### ISSUE THREAD

**Issue by naman-gupta99** [naman-gupta99/brahe-solar-system#2]

**Comment by heza-ru** [comment]
Recentered the circles you can review.

**Comment by naman-gupta99** [comment]
Need to recenter the complete animation to the center of the Page with all the small circles orbiting around the larger circle.

**Comment by heza-ru** [comment]
done! (In the new PR)

**Comment by naman-gupta99** [comment]
<img width="865" alt="image" src="https://user-images.githubusercontent.com/32611803/196630239-15f7ff2e-3205-4881-81a9-2c6838e88748.png">
Hey @heza-ru If you see here, the animation is in the center of the div but the div is not in the center of the page. Can you please check this? Also try and check if your solution is responsive or not. That is, does it work correctly on mobile phones and on different window sizes for desktop.

**Comment by heza-ru** [comment]
I set the while div as the center of the page that contains the solar system this is supposed to be responsive. Are you encountering any issues?

**Comment by naman-gupta99** [comment]
@heza-ru Yeah as you can see in the above screenshot it looks that the div is shifted to the bottom right corner for some reason. Also, if you check it in mobile device it looks weird there too.
